recipes/nbgrader/run_test.sh

```diff
@@ -0,0 +1,6 @@
+conda install --yes nodejs
```

Comment:
Add a `test/requires` section in the recipe instead of installing `nodejs` here.